lib/api/src/grpc/proto/snapshots_service.proto

```diff
@@ -58,6 +58,7 @@ message SnapshotDescription {
   string name = 1; // Name of the snapshot
   google.protobuf.Timestamp creation_time = 2; // Creation time of the snapshot
   int64 size = 3; // Size of the snapshot in bytes
+  string checksum = 4; // SHA256 digest of the snapshot file
```

Comment:
let's do this field optional, so it will be compatible with previous/next versions of the clients